pallets/duster/src/lib.rs

```diff
@@ -295,3 +295,14 @@ impl<T: Config> DustRemovalAccountWhitelist<T::AccountId> for Pallet<T> {
 		})
 	}
 }
+
+sp_api::decl_runtime_apis! {
```

Comment:
We usually define APIs in a separate crate. Check evm-accounts or currencies pallets.